neuror/axon.py

```diff
@@ -148,6 +148,12 @@ def repair(morphology, section, intact_sections, axon_branches, used_axon_branch
         appended = section.append_section(branch)
         translation = section.points[-1, COLS.XYZ] - appended.points[0, COLS.XYZ]
         align(appended, translation)
+
+        # Recalculate translation because align() sometimes does not *exactly* preserve the position
+        # of the first point. This causes translate() to not make a perfect translation and
+        # the consequence is that the appended section does not exactly starts at the end
+        # of the parent section. This in turns, causes a duplicate point when writing the morphology
+        translation = section.points[-1, COLS.XYZ] - appended.points[0, COLS.XYZ]
```

Comment:
not sure you gonna solve anything with this... 
You can still have decrepancies with this I guess due to float problems

Reply:
Now I manually set the parent section end point to the child section first point.
It solves it, I've checked by running the workflow with it.
But yes, that's crap.